Resources/MixedModels/Description.qml

```diff
@@ -3,50 +3,49 @@ import JASP.Module 	1.0
 
 Description
 {
-	title		: "Mixed Models"
+	title		: qsTr("Mixed Models")
 	icon		: "MixedModels_classical.svg"
-	description	: "This module offers the Linear Mixed Models analysis."
-	version		: "0.13"
```

Comment:
Version?

Reply:
:-( I fixed it with an amend. Have an appointment with the optician next week)